login-workflow/src/new-architecture/i18nInstance/index.ts

```diff
@@ -0,0 +1,53 @@
+import i18next from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+import { translations } from './translations';
+
+export const i18nBluiInstance = i18next
+    .createInstance(
+        {
+            lng: 'en',
+            detection: {
+                order: ['querystring', 'localStorage', 'navigator'],
+                caches: ['localStorage'],
+                lookupLocalStorage: 'blui-auth-i18nextLng',
+            },
+            fallbackLng: 'en',
+            ns: ['blui', 'app'],
```

Comment:
Refer to tom's implementation on the Registration side...there should only be a single instance and it should be something like bluiAuthentication